components/ProblemDetails.yml

```diff
@@ -0,0 +1,29 @@
+openapi: 3.0.0
+info:
+  version: "0.0.3-unstable"
+  title: VC API
+  description: This is an Experimental Open API Specification for the [VC Data Model](https://www.w3.org/TR/vc-data-model/).
+  license:
+    name: W3C Software and Document License
+    url: http://www.w3.org/Consortium/Legal/copyright-software.
+  contact:
+    name: GitHub Source Code
+    url: https://github.com/w3c-ccg/vc-api
+paths:
+components:
+  schemas:
+    ProblemDetails:
+      type:
+        type: string
+        description: URL identifying the type of problem.
+      title:
+        type: string
+        description: Short but specific human-readable string for the problem.
+      detail:
+        type: string
+        description: Longer human-readable string for the problem.
+      example: { 
+        "type": "https://www.w3.org/TR/vc-data-model#PARSING_ERROR", 
```

Comment:
I found it a bit confusing that this example was to a type that isn't actually defined in the VC data model as far as I can tell (a GET to `https://www.w3.org/TR/vc-data-model#PARSING_ERROR` just leads to the spec root for me). Are there actual error types defined for VC verification somewhere? If not, perhaps using `example.com` for now could be clearer that it is just an example.